### PR TITLE
Functional subscription and loading grave vault even if user not on latest forwarder

### PR DESCRIPTION
**Pull request by tehnalogos**
…atest forwarder

### CODE REVIEW

File: frontend/components/GraveContents.tsx [hunk 1]
```diff
@@ -16,38 +16,38 @@ import Link from 'next/link';
...
+        {graveTitle}
+      </Text>
+      {graveOwner === connectedAccount && (
+        <Link href="/grave/settings" passHref>
+          <Icon as={FaCog} color={'light.white'} h={5} w={6} />
+        </Link>
+      )}
+      <ShareButton pageAccount={graveOwner} />
+    </Flex>
+  );
```
Comment (democrazyfi) [inline frontend/components/GraveContents.tsx:46]: does this need to be extracted to a variable? it feels more readable inline below

File: frontend/components/GraveContents.tsx [hunk 2]
```diff
@@ -16,38 +16,38 @@ import Link from 'next/link';
...
+export default function GraveContents({ graveOwner }: { graveOwner: string }) {
   const walletContext = useContext(WalletContext);
-  const { account: connectedAccount, graveVault } = walletContext;
+  const { account: connectedAccount } = walletContext;
   let graveTitle = 'YOUR GRAVEYARD';
-  let pageGraveVault = null;
-  // account present and same as connected account
-  // account present but not same as connected account
-  if (connectedAccount && connectedAccount === account) {
+  if (connectedAccount && connectedAccount === graveOwner) {
```
Comment (democrazyfi) [inline frontend/components/GraveContents.tsx:23]: can be simplified to `connectedAccount === graveOwner`

File: frontend/components/GravePageAssets.tsx [hunk 3]
```diff
@@ -4,39 +4,52 @@ import { useContext, useEffect, useState } from 'react';
...
+  graveOwner,
 }: {
-  pageAccount: string;
-  pageGraveVault?: string | null;
+  graveOwner: string;
 }) {
   const walletContext = useContext(WalletContext);
   const { networkConfig } = walletContext;
-  const [graveVault, setGraveVault] = useState<string | null>(pageGraveVault);
+  const [graveVault, setGraveVault] = useState<string | null>(null);
```
Comment (Ledespana) [inline frontend/components/GravePageAssets.tsx:16]: should we worry about having it as null as the initial state?

Comment (tehnalogos) [reply frontend/components/GravePageAssets.tsx:16]: no because if it's loading (null) the code handles it for the UI

**Review by democrazyfi — APPROVED** [review]

**Pull request merged**